aries-core/src/js/components/core/FormField/FormField.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { FormField as GrommetFormField } from 'grommet';
+
+const StyledFormField = styled(GrommetFormField)`
+  ${props =>
+    props.visuallyHidden &&
+    `label {
+      border: 0;
+      clip: rect(0 0 0 0);
+      height: 1px;
+      margin: -1px;
+      overflow: hidden;
+      padding: 0;
+      position: absolute;
+      width: 1px;
+    }`};
+`;
+
+export const FormField = ({ visuallyHidden, ...rest }) => {
```

Comment:
wondering if the prop should be called `labelHidden` or `hiddenLabel` since it's not the whole formfield becoming visually hidden, but just the label right?

Also, hesitant to pull this into the aries-core now that teams may be using the design system. I'm wondering if this prop would be valuable to have in grommet as well. If so, this wouldn't cause any import errors from teams (or us) having to swap out every import instance. Maybe it'd be good to discuss with @ericsoderberghp and see if a prop like this would have value in Grommet and if so we could implement there.

Reply:
Agreed on putting this in grommet.

What about: `label={{ text, hidden }}`, to make it clear it's about `label`?

Reply:
I like that.

Reply:
Agree. This was intended as more of a way to propose an approach. I can move it to be an `aries-site` component to not muddy `aries-core`.

Reply:
> wondering if the prop should be called `labelHidden` or `hiddenLabel` since it's not the whole formfield becoming visually hidden, but just the label right?

good call.

Reply:
👍